taxcalc/behavior.json

```diff
@@ -53,7 +59,10 @@
         "boolean_value": true,
         "integer_value": true,
         "value": [false],
-        "validations": {"min": false, "max": true}
+        "range": {"min": false, "max": true},
+        "out_of_range_minmsg": "",
+        "out_of_range_maxmsg": "",
+        "out_of_range_action": "stop"
     },
```

Comment:
Should this parameter be boolean?

Reply:
Which parameter are you talking about?

Reply:
Sorry about that. I'm referring to `_BE_subinc_wrt_earnings`.